tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,374 @@
+# Preprocessing
+# ^^^^^^^^^^^^^
+#
+# MNE-Python supports a variety of preprocessing approaches and techniques
+# (maxwell filtering, signal-space projection, independent components analysis,
+# filtering, downsampling, etc); see the full list of capabilities in the
+# :mod:`mne.preprocessing` and :mod:`mne.filter` submodules. Here we'll clean
+# up our data by performing independent components analysis
+# (:class:`~mne.preprocessing.ICA`); for brevity we'll skip the steps that
+# helped us determined which components best capture the artifacts (see
+# :doc:`../preprocessing/plot_artifacts_correction_ica` for a detailed
+# walk-through of that process).
+
+# set up and fit the ICA
+ica = mne.preprocessing.ICA(n_components=20, random_state=97, max_iter=800)
+ica.fit(raw)
+ica.exclude = [1, 2]  # details on how we picked these are omitted here
+ica.plot_properties(raw, picks=ica.exclude)
+
+###############################################################################
+# Once we're confident about which component(s) we want to remove, we pass them
+# as the ``exclude`` parameter and then apply the ICA to the raw signal. The
+# :meth:`~mne.preprocessing.ICA.apply` method requires the raw data to be
+# loaded into memory (by default it's only read from disk as-needed), so we'll
+# use :meth:`~mne.io.Raw.load_data` first. We'll also make a copy of the
+# :class:`~mne.io.Raw` object so we can compare the signal before and after
+# artifact removal side-by-side:
+
+orig_raw = raw.copy()
+raw.load_data()
+ica.apply(raw)
+
+# show some frontal channels to clearly illustrate the artifact removal
+chs = ['MEG 0111', 'MEG 0121', 'MEG 0131', 'MEG 0211', 'MEG 0221', 'MEG 0231',
+       'MEG 0311', 'MEG 0321', 'MEG 0331', 'MEG 1511', 'MEG 1521', 'MEG 1531',
+       'EEG 001', 'EEG 002', 'EEG 003', 'EEG 004', 'EEG 005', 'EEG 006',
+       'EEG 007', 'EEG 008']
+chan_idxs = [raw.ch_names.index(ch) for ch in chs]
+orig_raw.plot(order=chan_idxs, start=12, duration=4)
+raw.plot(order=chan_idxs, start=12, duration=4)
+
+###############################################################################
+# Detecting experimental events
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# First extract events:
+# The sample dataset includes several "STIM" channels that recorded electrical
+# signals sent from the stimulus delivery computer (as brief DC shifts /
+# squarewave pulses). These pulses (often called "triggers") are used in this
+# dataset to mark experimental events: stimulus onset, stimulus type, and
+# participant response (button press). The individual STIM channels are
+# combined onto a single channel, in such a way that voltage
+# levels on that channel can be unambiguously decoded as a particular event
+# type. On older Neuromag systems (such as that used to record the sample data)
+# this summation channel was called ``STI 014``, so we can pass that channel
+# name to the :func:`mne.find_events` function to recover the timing and
+# identity of the stimulus events.
```

Comment:
I would point to `:term:stim channel` somewhere in this paragraph. (Do we explain somewhere the different types of stim channels people can find?)